packages/api-fetch/src/middlewares/user-locale.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+function userLocaleMiddleware( options, next ) {
+	if (
+		typeof options.url === 'string' &&
+		-1 === options.url.indexOf( '?_locale=' ) &&
+		-1 === options.url.indexOf( '&_locale=' )
```

Comment:
It'd be nice if `@wordpress/url` had a `hasQueryArg` helper... Not critical for this PR though.

Reply:
@danielbachhuber Ha :-) I thought the same and created #10879 because of this.